phpunit.xml

```diff
@@ -11,6 +11,7 @@
   >
   <php>
     <ini name="error_reporting" value="-1" />
+    <env name="DATABASE_URL" value="sqlite:///:memory:" />
```

Comment:
Unit tests should not rely on any database. phpunit.xml is dedicated to unit tests only

Reply:
Fixed in fdfec9f0.